stock_warehouse_flow_release/models/stock_move.py

```diff
@@ -8,13 +8,6 @@
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    def _apply_flow_on_action_confirm(self):
-        # Override to not apply the flow configuration on moves to release.
-        # The flow will be applied on the release, not before.
-        if self.rule_id.route_id.available_to_promise_defer_pull:
-            return False
-        return super()._apply_flow_on_action_confirm()
-
```

Comment:
With this change it is called on confirm when the flag `disable_warehouse_flow_on_confirm` isn't set and called on release. Shouldn't it be checked on release the now if the flag is set or not. 
Why is the flag set by the migration but default turned on for routes with `available_to_promise_defer_pull`?

Reply:
It has been moved in `stock_warehouse_flow`
https://github.com/OCA/wms/pull/991/files#diff-531a3523042e74031943fc83a9176f2de25e3ac2be78e85d1edb27f92ecd669cR19-R23
We now have a dedicated option for this, rather than checking if `available_to_promise_defer_pull` is set on the route.
We also have a migration for this.

Reply:
I think the `disable_warehouse_flow_on_confirm` should move into `stock_warehouse_flow_release` because only there you want to decide if you want to calc the flow on confirm or on release.
If you just want to disable the calc of the flow in the base addon you just don't need to configure it.
But thats not a blocker for me. 

But if you still want to keep `disable_warehouse_flow_on_confirm` on the base module, then i think there should be also a dedicated flag to allow the flow calculation on release. 

Otherwise the flow will be calculated always on release. 

In my opinon we should either 
- add a dedicated flag for this
or
- keep the addon as it is just removing the `auto_install` flag

cc @jbaudoux

Reply:
https://github.com/OCA/wms/pull/991#discussion_r2012056885

Reply:
done